src/components/common/Actions.vue

```diff
@@ -207,6 +221,26 @@ export default class Actions extends Mixins(DateMixin, FilingTemplateMixin, Lega
     }
   }
 
+  /** Fetches NR and validates it. */
+  private async validateNameRequest (nrNumber: string): Promise<void> {
```

Comment:
I noticed this is similar to processNameRequest() in App.vue. Can we name both functions the same? (I like this function's name better.)

Reply:
The other one has more things to it like setting the data to the store and all. This is just validating the NR

Reply:
I was hoping to reconcile the 2 functions that do something very similar, by naming them the same. (I realize they do slightly different things so they can be merged.) It just doesn't feel very DRY at this point.
How about adding a comment to each function saying it's similar to the other one?